packages/devextreme/js/data/store.d.ts

```diff
@@ -0,0 +1,209 @@
+import {
+  DxPromise,
+  DxExtendedPromise,
+} from '../core/utils/deferred';
+import { DeepPartial } from '../core/index';
+import {
+  FilterDescriptor,
+  GroupDescriptor,
+  LoadOptions,
+} from './index';
+
+export type Options<
+    TItem = any,
+    TKey = any,
+> = StoreOptions<TItem, TKey>;
+
+/**
+ * @docid
+ * @namespace DevExpress.data
+ * @deprecated Use Options instead
+ * @hidden
+ */
+export interface StoreOptions<
```

Comment:
Since we're moving StoreOptions to a new location, there is no need to keep back.compat. reexports.

```suggestion
export type Options<
```

Reply:
You have to add values to docid tags then **SIGH**

Reply:
Fixed. Remove StoreOptions. Appended the StoreOptions. prefix to a type and all its members.